src/biteme/__init__.py

```diff
@@ -1,5 +1,88 @@
-from biteme.__main__ import cli
-from biteme.pybites import _bite_info, download_bite
+from __future__ import annotations
```

Comment:
I am not a big fan of putting code in `__init__.py` vs using a named module (more explicit). Any advantage doing it this way?

Reply:
I want a `py.typed` file distributed because I ❤️ annotations but you can't distributed single modules with a `py.typed` file. So having `__init__.py` only is the next best thing. I don't think it _really_ warrants a package just yet.

Reply:
Ah I now I remember you told me that. Does breaking out cli into its own module ruin this plan?

Reply:
Nope, not at all.